CMake fails on the longturn.net server

**Describe the bug**

```
CMake Error at cmake/FreecivDependencies.cmake:97 (target_include_directories):                                                                                                                                                             
  target_include_directories called with non-compilable target type                                                                                                                                                                         
Call Stack (most recent call first):                                                                                                                                                                                                        
  CMakeLists.txt:54 (include) 
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install Debian buster and fc21 dependencies
2. Do `mkdir build; cmake -B build .`
3. See error in CMake output

**Expected behavior**

Can configure and build without problems.

**Platform and version (please complete the following information):**
 - OS: Debian Buster
 - Freeciv21 version: 3.0-alpha5

**Additional context**

Buster has CMake 3.13 which should work fine with the syntax being used, but appears not to. Need to downgrade to the older way:
```cmake
set_target_properties(lua PROPERTIES
  IMPORTED_LOCATION "${loc}"
  INTERFACE_INCLUDE_DIRECTORIES "${LUA_INCLUDE_DIR}"
  INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${LUA_INCLUDE_DIR}")
```

--- a/cmake/FreecivDependencies.cmake
+++ b/cmake/FreecivDependencies.cmake
@@ -91,10 +91,11 @@ find_package(Lua 5.3 REQUIRED)
 add_library(lua UNKNOWN IMPORTED GLOBAL)
 # Get a library name for IMPORTED_LOCATION
 list(GET LUA_LIBRARIES 0 loc)
-set_target_properties(lua PROPERTIES IMPORTED_LOCATION "${loc}")
+set_target_properties(lua PROPERTIES
+  IMPORTED_LOCATION "${loc}"
+  INTERFACE_INCLUDE_DIRECTORIES "${LUA_INCLUDE_DIR}")
 # Link to all libs, not just the first
 target_link_libraries(lua INTERFACE "${LUA_LIBRARIES}")
-target_include_directories(lua SYSTEM INTERFACE "${LUA_INCLUDE_DIR}")
 
 if (CMAKE_CROSSCOMPILING AND NOT CMAKE_CROSSCOMPILING_EMULATOR)
   find_package(ToLuaProgram REQUIRED)

